LayoutTest.xcodeproj/xcshareddata/xcschemes/LayoutTestTests.xcscheme

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It looks like you've added LayoutTestBase and LayoutTestTests. Why not LayoutTest? And why do you need to make a test schema public?